index.js

```diff
@@ -374,6 +374,8 @@ function attributeValueToJSXPropValue(key, value) {
 
       return styles;
     }, {});
+  } else if (key === 'href') {
```

Comment:
is this needed for `src` too on image tags for example?

Reply:
Good call, I believe so. I’ll check and add support for this if needed.

Reply:
Actually, it looks like only ≤ IE6 are affected by this (modern browsers don't execute javascript-scheme `src` attrs), and since ReactDOM doesn't work on IE6, I think we're in the clear.

Reply:
Ok but we still need it for href?

Reply:
Yep; React will gleefully pass these through and browsers interpret them.